### ISSUE TITLE
Question: What Encoder Does Aves Use to Encode JPEGs?

### ISSUE THREAD

**Issue by OkyDooky** [deckerst/aves#677]
Is it MozJPEG or something different? In many benchmarks online, MozJPEG performs as good if not better than WebP.

**Comment by deckerst** [comment]
Aves uses the native Android encoders for images, not an external library.

**Comment by OkyDooky** [comment]
Okay. I checked and it seems like Android just uses the basic libjpeg encoder. :/
However, I looked a bit more and it seems that there is already some efforts making mozjpeg available in Flutter apps, like
https://pub.dev/packages/flutter_mozjpeg/example
and
https://github.com/espresso3389/flutter_mozjpeg

So, my next question would be: should I open a feature request to support (even optionally) of the mozjpeg encoder in Aves? Or is that something you have less than zero interest in implementing? It vaguely seems doable and it would definitely improve the output quality (only drawback is it would be slower, especially on older devices, but supposedly not more than the natibe WebP encoder).
